dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -914,18 +914,30 @@ ssid=Subsystem ID
 ssid.desc=JDBC driver property: ssid.
 ssl=SSL
 ssl.desc=JDBC driver property: ssl.
+sslfactory=SSL factory class name
+sslfactory.desc=JDBC driver property: sslfactory.
+sslCert=SSL certificate location
+sslCert.desc=JDBC driver property: sslCert.
 sslCertLocation=SSL certificate location
 sslCertLocation.desc=JDBC driver property: sslCertLocation.
 SSLCipherSuites=SSL cipher suites
 SSLCipherSuites.desc=JDBC driver property: SSLCipherSuites.
 sslConnection=SSL connection
 sslConnection.desc=JDBC driver property: sslConnection.
+sslKey=SSL key location
+sslKey.desc=JDBC driver property: sslKey.
 sslKeyStoreLocation=SSL key store location
 sslKeyStoreLocation.desc=JDBC driver property: sslKeyStoreLocation.
 sslKeyStorePassword=SSL key store password
 sslKeyStorePassword.desc=JDBC driver property: sslKeyStorePassword.
 sslKeyStoreType=SSL key store type
 sslKeyStoreType.desc=JDBC driver property: sslKeyStoreType.
+sslMode=SSL mode
+sslMode.desc=JDBC driver property: sslMode.
+sslPassword=SSL password
+sslPassword.desc=JDBC driver property: sslPassword.
+sslRootCert=SSL root certificate location
+sslRootCert=JDBC driver property: sslRootCert
```

Comment:
Based on the format of the rest of this file, should 
"sslRootCert=JDBC driver property: sslRootCert"
be
"sslRootCert.desc=JDBC driver property: sslRootCert."?

Reply:
good catch, will fix this